pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2818,6 +2819,30 @@ func handlePhysicalIOAdapterListImpl(ctxArg interface{}, key string,
 					}
 					aa.AddOrUpdateIoBundle(log, vfIb)
 				}
+			} else if ib.Type == types.IoVCAN {
+				// Initialize (create and enable) Virtual CAN device
+				vcan, err := canbus.AddVCANLink(ib.Ifname)
+				if err != nil {
+					log.Errorf("Failed to add Virtual CAN interface: %s", err)
+				}
+				err = canbus.LinkSetUp(vcan)
+				if err != nil {
+					log.Errorf("Failed to enable virtual CAN interface: %s", err)
```

Comment:
```suggestion
					err := fmt.Errorf("Failed to enable virtual CAN interface: %s", err)
					log.Error(err)
					ib.Error = err
					ib.ErrorTime = time.Now()
```
then the user in the cloud can see directly the error message